as-vending/res/configuration.toml

```
@@ -66,7 +66,9 @@ Type="edgex-messagebus"
   ControllerBoardDisplayRow3Cmd = "displayRow3"
   ControllerBoardLock1Cmd = "lock1"
   ControllerBoardLock2Cmd = "lock2"
-  DeviceNames   = "ds-card-reader,Inference-MQTT-device"
+  CardReaderDeviceName   = "card-reader"
+  MQTTDeviceName = "Inference-MQTT-device"
```

Shouldn't have the protocol in the device name. From the app level we don't care which protocol... Name will also need to be changed in the Device Definition toml file. 
```suggestion
  InferenceDeviceName = "Inference-device"
```